### PR TITLE
[PRT] Port 8.0 of web_ir_actions_act_window_page

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

File: web_ir_actions_act_window_page/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Client side paging
+=====================
```
Comment (pedrobaeza) [inline web_ir_actions_act_window_page/README.md]: Use same length of the title

File: web_ir_actions_act_window_page/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+This addon enables buttons to return::
```
Comment (pedrobaeza) [inline web_ir_actions_act_window_page/README.md]: buttons? Isn't it client side actions?

File: web_ir_actions_act_window_page/__openerp__.py [hunk 3]
```diff
@@ -19,39 +19,14 @@
...
-    ],
-    "qweb": [
-    ],
-    "auto_install": False,
-    'installable': False,
-    "external_dependencies" : {
-        'python' : [],
-    },
+    "name": "Window actions for client side paging",
+    "summary": ("Allows a developer to trigger a pager to show the previous "
```
Comment (pedrobaeza) [inline web_ir_actions_act_window_page/__openerp__.py:23]: This summary is very large and will not fit correctly in module kanban view.

**Comment by legalsylvain** [comment]
Thanks. 
:+1: (Code Review, no test.)

File: web_ir_actions_act_window_page/README.md (hunk 2 above)
Comment (StefanRijnhart) [inline web_ir_actions_act_window_page/README.md]: Don't know. We use it for buttons, when performing some action + move to next record.

File: web_ir_actions_act_window_page/README.md (hunk 1 above)
Comment (StefanRijnhart) [inline web_ir_actions_act_window_page/README.md]: Done, thanks!

File: web_ir_actions_act_window_page/__openerp__.py (hunk 3 above)
Comment (StefanRijnhart) [inline web_ir_actions_act_window_page/__openerp__.py:23]: Done!

File: web_ir_actions_act_window_page/README.md (hunk 2 above)
Comment (pedrobaeza) [inline web_ir_actions_act_window_page/README.md]: Yeah, but you use it as a return value for the method of a button to perform a client side action

Comment (StefanRijnhart) [inline web_ir_actions_act_window_page/README.md]: See your point, updated the description.

**Comment by pedrobaeza** [comment]
Thank you very much.

:+1: when Travis presumibly give green.

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**